src/WordSearch/Point.ts

```diff
@@ -0,0 +1,8 @@
+export class Point {
+  x: number;

```

Comment:
Should these be mutable?  Would it be worth adding `readonly` to at least harden the TypeScript surface?